examples/with-svelte/client/README.md

```diff
@@ -0,0 +1,107 @@
+_Psst — looking for a more complete solution? Check out [SvelteKit](https://kit.svelte.dev), the official framework for building web applications of all sizes, with a beautiful development experience and flexible filesystem-based routing._
```

Comment:
The name of the folder for this app should be changed to `with-svelte-react-thirdpartyemailpassword`